### PR TITLE
[docs] rework storage struct how-to

### PR DESCRIPTION
**Pull request by Eve-Parity**
Need some "before you begin" type of material. i.e. what do you need to know/have configured/etc...

### CODE REVIEW

File: v3/how-to-guides/02-pallet-design/e-storage-value/index.mdx [hunk 1]
```diff
@@ -8,124 +8,112 @@ category: pallet design
...
-```rust
-// Declare `admin` as type `T::AccountId`.
-#[pallet::genesis_config]
-pub struct GenesisConfig<T: Config> {
-	pub admin: T::AccountId,
+Creating a struct of similarly grouped storage items is an orderly way to keep track of them. 
+When grouped in this way, it is easier to reference them than it is if individual `StorageValue` items are kept separately.
+This can make testing and genesis configuration easier.
+
+This guide shows you how to create a [`StorageValue`](/rustdocs/latest/frame_support/storage/trait.StorageValue.html) item that holds a struct and is used in [`on_initialize`](/rustdocs/latest/frame_support/traits/trait.Hooks.html#method.on_initialize).
```
Comment (sacha-l) [inline v3/how-to-guides/02-pallet-design/e-storage-value/index.mdx:15]:
```suggestion
This guide shows you how to create a [`StorageValue`](/rustdocs/latest/frame_support/storage/trait.StorageValue.html) storage item that holds a struct and is used in [`on_initialize`](/rustdocs/latest/frame_support/traits/trait.Hooks.html#method.on_initialize). 
```

File: v3/how-to-guides/02-pallet-design/e-storage-value/index.mdx [hunk 2]
```diff
@@ -8,124 +8,112 @@ category: pallet design
...
+		let value: T::Balance = 50u8.into();
+		meta.issuance = meta.issuance.saturating_add(value);
 
-}
-```
+		// Add the amount to the `minter` account in storage.
+		Accounts::<T>::mutate(&meta.minter, |bal| {
+			*bal = bal.saturating_add(value);
+		});
+
```
Comment (sacha-l) [inline v3/how-to-guides/02-pallet-design/e-storage-value/index.mdx:108]:
```suggestion
```

File: v3/how-to-guides/02-pallet-design/f-tight-coupling/index.mdx [hunk 3]
```diff
@@ -7,30 +7,14 @@ category: pallet design
...
-    },
-  ]}
-/>
-
-## Steps
-
-### 1. Configure your workspace
+This guide will show you how to use an external pallet's capabilities from inside your pallet.
+
+Tight coupling two pallets may suit specific use cases where an outside pallet contains many types and methods common to those of your initial pallet. If this is not the case, you can use loose coupling instead. To learn more, see [pallet coupling](/v3/runtime/pallet-coupling).
```
Comment (sacha-l) [inline v3/how-to-guides/02-pallet-design/f-tight-coupling/index.mdx]:
```suggestion
Tight coupling two pallets may suit specific use cases where an outside pallet contains many types and methods common to those of your initial pallet. 
If this is not the case, you can use loose coupling instead. 
To learn more, see [pallet coupling](/v3/runtime/pallet-coupling).
```

File: v3/how-to-guides/02-pallet-design/f-tight-coupling/index.mdx [hunk 4]
```diff
@@ -7,58 +7,47 @@ category: pallet design
...
-
-```rust
-// Get the members from the vec-set pallet
-let who = special_pallet::Pallet::<T>::get();
-```
-
-The above snippet assumes that `special_pallet` contains a method called `get()`.
+This guide will show you how to use a method from an external pallet inside your pallet.
+
+Tight coupling two pallets may suit specific use cases where an outside pallet contains many types and methods common to those of your initial pallet.
```
Comment (lisa-parity) [inline v3/how-to-guides/02-pallet-design/f-tight-coupling/index.mdx:12]: I suggest starting with a brief (one sentence-ish) definition of what "tight coupling" means even if it is redundant to information that's somewhere else. Then something like (i'm not sure what you mean by external and outside--does it mean not part of the runtime?):

Tight coupling between pallets can be useful when one pallet requires many of the same types and methods that are defined in another pallet. 
This guide illustrates tight coupling by showing you how to use a method from an external pallet from within your current pallet.

Again I'm not sure about the intention of external, so I'm guessing a little.

File: v3/how-to-guides/02-pallet-design/f-tight-coupling/index.mdx [hunk 5]
```diff
@@ -7,58 +7,47 @@ category: pallet design
...
-```rust
-// Get the members from the vec-set pallet
-let who = special_pallet::Pallet::<T>::get();
-```
-
-The above snippet assumes that `special_pallet` contains a method called `get()`.
+This guide will show you how to use a method from an external pallet inside your pallet.
+
+Tight coupling two pallets may suit specific use cases where an outside pallet contains many types and methods common to those of your initial pallet. 
+If this is not the case, you can use loose coupling instead.
```
Comment (lisa-parity) [inline v3/how-to-guides/02-pallet-design/f-tight-coupling/index.mdx:13]:
```suggestion
```

File: v3/how-to-guides/02-pallet-design/f-tight-coupling/index.mdx [hunk 6]
```diff
@@ -7,58 +7,47 @@ category: pallet design
...
-// Get the members from the vec-set pallet
-let who = special_pallet::Pallet::<T>::get();
-```
-
-The above snippet assumes that `special_pallet` contains a method called `get()`.
+This guide will show you how to use a method from an external pallet inside your pallet.
+
+Tight coupling two pallets may suit specific use cases where an outside pallet contains many types and methods common to those of your initial pallet. 
+If this is not the case, you can use loose coupling instead. 
+To learn more, see [pallet coupling](/v3/runtime/pallet-coupling).
```
Comment (lisa-parity) [inline v3/how-to-guides/02-pallet-design/f-tight-coupling/index.mdx:14]:
```suggestion
To learn more about tight coupling and loose coupling between pallets, see [pallet coupling](/v3/runtime/pallet-coupling).
```

**Review by lisa-parity — APPROVED** [review]
A few suggestions. I think the first How-to could easily be fully incorporated into the code using comments as we discussed a little bit as a future direction (where the code and documentation are one thing from a maintenance perspective).

**Pull request merged**